duckduckgo/macos-browser
# DBP UI Middleware

Task/Issue URL: https://app.asana.com/0/72649045549333/1205205203094796/f
Tech Design URL: https://app.asana.com/0/481882893211075/1205250208033190/f
CC: @Bunn @jotaemepereira 

**Description**:
This PR adds a middleware layer to communicate with the DBP UI

Web UI is in progress. For the moment I have set up a debug page to test this PR.

**Steps to test this PR**:
1. Open Data Broker Protection
2. You should be presented with a white page with a few buttons
3. Attach a debugger to the webview either through Safari or by directly opening dev tools
4. Clicking either of the top two buttons on the page should display a log in the Xcode console
5. Clicking the button below these should show a log for `missing 'setState'` in the web console (this is correct)



---
###### Internal references:
[Pull Request Review Checklist](https://app.asana.com/0/1202500774821704/1203764234894239/f)
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
[Pull Request Documentation](https://app.asana.com/0/1202500774821704/1204012835277482/f)
